components/item-full.js

```diff
@@ -188,7 +195,7 @@ function TopLevelItem ({ item, noReply, ...props }) {
           !item.deletedAt &&
           // Don't show related items for items with bounties, show past bounties instead
           !(item.bounty > 0) &&
-            <Related title={item.title} itemId={item.id} show={item.ncomments === 0} />
+            <Related title={item.title} itemId={item.id} show={item.ncomments % 2 === 0} />
```

Comment:
Was this only for testing? This makes the related section show for posts with even comment count

Reply:
Oh bugger. Yes that was for testing. I'll put that back. It saved me from making new posts each time and waiting 10 mins

Reply:
Done